packages/RNTester/js/RNTesterApp.ios.js

```diff
@@ -74,7 +74,7 @@ const Header = ({
             {onBack && (
               <View>
                 <Button
-                  title="Back"
+                  title="BackBOO"
```

Comment:
All looks good to me 🚀 , but I believe this is a typo?

Reply:
Oh rip yeah, that was a live refresh testing remnant. Will fix that right now. Thanks!